torchdynamo/convert_frame.py

```diff
@@ -72,6 +72,9 @@ def clear(self):
 output_codes = Tracker()
 
 
+initial_grad_state = None
```

Comment:
Ideally we wouldn't use a global for this.

Reply:
I'll address this as follow-up, thanks!